workflows/maskrcnn-training/template.yaml

```diff
@@ -85,18 +87,12 @@ templates:
     - mountPath: /mnt/output
       name: output
     workingDir: /mnt/src
-  nodeSelector:
-    beta.kubernetes.io/instance-type: '{{workflow.parameters.sys-node-pool}}'
```

Comment:
We can put this back in, the new API supports these properly now.

Reply:
ok, updated templates.